source/frontend/src/features/mapSideBar/acquisition/add/AddAcquisitionForm.tsx

```diff
@@ -133,9 +136,23 @@ const AddAcquisitionDetailSubForm: React.FC<{
   const acquisitionTypes = getOptionsByType(API.ACQUISITION_TYPES);
   const acquisitionPhysFileTypes = getOptionsByType(API.ACQUISITION_PHYSICAL_FILE_STATUS_TYPES);
   const acquisitionFundingTypes = getOptionsByType(API.ACQUISITION_FUNDING_TYPES);
+  const subfileInterestTypes = getOptionsByType(API.SUBFILE_INTERETST_TYPES);
 
   const isSubFile = exists(parentId) && isValidId(parentId);
 
+  const {
```

Comment:
Hmm, This is just a re-order of the existing code right?

Reply:
Yes